omod/src/main/java/org/openmrs/web/controller/PortletController.java

```diff
@@ -265,6 +265,34 @@ public ModelAndView handleRequest(HttpServletRequest request, HttpServletRespons
 					} else {
 						model.put("patientObs", new HashSet<Obs>());
 					}
+					// information about whether or not the patient has exited care
```

Comment:
Can you comment on whether this is copied back in from somewhere, and if so, where?